pkg/kv/store.go

```diff
@@ -14,13 +14,14 @@ import (
 )
 
 const (
-	InitialMigrateVersion    = 1
-	ACLMigrateVersion        = 2
-	ACLNoReposMigrateVersion = 3
-	ACLImportMigrateVersion  = 4
-	LatestVersion            = ACLImportMigrateVersion
-	PathDelimiter            = "/"
-	MetadataPartitionKey     = "kv-internal-metadata"
+	InitialMigrateVersion = iota + 1
+	ACLMigrateVersion
+	ACLNoReposMigrateVersion
+	ACLImportMigrateVersion
+	LatestVersion = ACLImportMigrateVersion
```

Comment:
can remove the last assignment (call it something like NextMigrateVersion) and use a function that will compare it with NextMigrateVersion-1 to see if it is latest.